packages/samples/angular/.eslintrc.json

```diff
@@ -3,5 +3,8 @@
   "parser": "@typescript-eslint/parser",
   "plugins": ["@typescript-eslint"],
   "extends": ["./node_modules/gts"],
-  "ignorePatterns": ["**/*.js", "dist"]
+  "ignorePatterns": ["**/*.js", "dist"],
+  "rules": {
+    "node/no-unpublished-import": "off"
```

Comment:
The root `package.json` has `private: true`, which triggers this error when importing `import {workspacesRoot} from '../../../../scripts/packages.mjs';`.